### ISSUE TITLE
Improve PWA designsystem site features

### ISSUE THREAD

**Issue by adamzerella** [govau/design-system-site#290]
# Bug report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/designsystem/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/designsystem/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

It seems that the current designsystem site isn't very progressive, there's a few things we can look at:

- [ ] Add `/robots.txt` to production environment, this was added for staging/testing with the recent `nginx_buildpack` addition.
- [ ] Add `/sitemap.xml` this is great for SEO and users looking for a perfect reference of site pages.
- [x] Add service worker to improve asset caching and offline access
- [x] Missing `/manifest.json`
- [x] Missing `<meta description=""/>` element
- [ ] `aria-*` attributes do not match their role, see:
![image](https://user-images.githubusercontent.com/1501560/47476006-e7fbba80-d869-11e8-9b27-1738b83cfe04.png) and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

This issue should be separate into an epic or individual tasks at the least.

**Comment by adamzerella** [comment]
Closing this as we have other separate issues to address this stuff, e.g #461 #487 #306
